app/vmalert/main.go

```diff
@@ -37,17 +37,18 @@ absolute path to all .yaml files in root.`)
 	basicAuthUsername = flag.String("datasource.basicAuth.username", "", "Optional basic auth username for -datasource.url")
 	basicAuthPassword = flag.String("datasource.basicAuth.password", "", "Optional basic auth password for -datasource.url")
 
-	remoteWriteURL = flag.String("remotewrite.url", "", "Optional URL to Victoria Metrics or VMInsert where to persist alerts state"+
+	remoteWriteURL = flag.String("remoteWrite.url", "", "Optional URL to Victoria Metrics or VMInsert where to persist alerts state"+
 		" in form of timeseries. E.g. http://127.0.0.1:8428")
-	remoteWriteUsername = flag.String("remotewrite.basicAuth.username", "", "Optional basic auth username for -remotewrite.url")
-	remoteWritePassword = flag.String("remotewrite.basicAuth.password", "", "Optional basic auth password for -remotewrite.url")
+	remoteWriteUsername     = flag.String("remoteWrite.basicAuth.username", "", "Optional basic auth username for -remotewrite.url")
+	remoteWritePassword     = flag.String("remoteWrite.basicAuth.password", "", "Optional basic auth password for -remotewrite.url")
+	remoteWriteMaxQueueSize = flag.Int("remoteWrite.maxQueueSize", 100, "Optional remote write maxQueueSize, default 100, you need to make it bigger if push print err: `failed to push timeseries to remotewrite: failed to push timeseries - queue is full`")
```

Comment:
I'd suggest following:
* description - `Defines the max number of pending datapoints to remote write endpoint.`
* default value - 10000; this should be enough for 99% of cases and doesn't look expensive in terms of memory
* drop `you need to make it bigger if push print err: `failed to push timeseries to remotewrite: failed to push timeseries - queue is full` hint from description and add recommendation to increase `remoteWrite.maxQueueSize` value into error message itself.

Reply:
ok